_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -76,11 +76,11 @@ Use the following options when customizing the `s3` sink.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-`bucket` | Yes | String | The name of the S3 bucket to which the sink writes.
-`codec` | Yes | [Codec](#codec) | The codec that determines how the data is serialized in the S3 object.
+`bucket` | Yes | String | The object from which the data is retrieved and then stored. The `name` must match the name of your object store.
+`codec` | Yes | [Codec](#codec) | The format of the output data.
 `aws` | Yes | AWS | The AWS configuration. See [aws](#aws) for more information.
 `threshold` | Yes | [Threshold](#threshold-configuration) | Configures when to write an object to S3. 
-`object_key` | No | [Object key](#object-key-configuration) | Sets the `path_prefix` of the object in S3. Defaults to the S3 object `events-%{yyyy-MM-dd'T'hh-mm-ss}` found in the root directory of the bucket.
+`object_key` | No | Sets the `path_prefix` and the `file_pattern` of the object store. The file pattern is always `events-%{yyyy-MM-dd'T'hh-mm-ss}` found inside the root directory of the bucket. The `path_prefix` is configurable.
```

Comment:
```suggestion
`object_key` | No | Sets the `path_prefix` and the `file_pattern` of the object store. The file pattern is always `events-%{yyyy-MM-dd'T'hh-mm-ss}`. By default, those objects are found inside the root directory of the bucket. The `path_prefix` is configurable.
```

Using the `path_prefix` will make the objects show up in somewhere that is not a "root" directory. So I tried to clarify that.